packages/decap-cms-core/src/components/Editor/EditorPreviewPane/EditorPreviewPane.js

```diff
@@ -261,7 +280,7 @@ PreviewPane.propTypes = {
 
 function mapStateToProps(state) {
   const isLoadingAsset = selectIsLoadingAsset(state.medias);
-  return { isLoadingAsset, config: state.config };
+  return { isLoadingAsset, config: state.config, state };
```

Comment:
It is needed for the implementation. It is passed into further functions like tryLoadEntry.

I'm not a 100% if this is the way to go though, it's been a while since I worked with redux, but using a thunk might be a better idea?

Reply:
Ah never mind then, all good :) Will be released today!